backend/src/db/migrations/20230906150722_make_users_email_index.ts

```diff
@@ -0,0 +1,13 @@
+import { type Knex } from 'knex';
+
+async function up(knex: Knex): Promise<void> {
+  return await knex.raw(
+    'CREATE UNIQUE INDEX users_lowercase_email ON users (lower(email));',
+  );
+}
+
+async function down(knex: Knex): Promise<void> {
+  return await knex.raw('DROP index IF EXISTS users_lowercase_email');
```

Comment:
Initially it has just unique constraint, so I believe it also should be moved back on rollback

Reply:
I think DROP INDEX also removes UNIQUE constraint if I uderstood you correctly. 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/44391531/57f441b7-4140-4f84-bcd5-d1ab430c8c57)

Reply:
I mean drop initial unique constraint on migration,, and turn it back on migration rollback